### PR TITLE
3.4 updates

### PR DESCRIPTION
**Pull request by trp89**

### CODE REVIEW

**Comment by trp89** [comment]
@ctgraham and/or @wopsononock I just wanted another quick onceover to make sure these are good to merge the pull request

File: InlineHtmlGalleySettingsForm.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+namespace APP\plugins\generic\inlineHtmlGalley;
+
+use PKP\form\Form;
+use PKP\form\validation\FormValidatorPost;
+use PKP\form\validation\FormValidatorCSRF;
+use APP\template\TemplateManager;
+
+class InlineHtmlGalleySettingsForm extends Form {
+
+	/** @var $plugin InlineHtmlGalleyPlugin */
```
Comment (ctgraham) [inline InlineHtmlGalleySettingsForm.php:23]: This file was committed with tabs, but PKP has standardized on spaces.

File: InlineHtmlGalleySidebarBlockPlugin.php [hunk 2]
```diff
@@ -11,14 +11,15 @@
...
  */
+namespace APP\plugins\generic\inlineHtmlGalley;
 
-import('plugins.generic.inlineHtmlGalley.InlineHtmlGalleyBlockPlugin');
+use APP\plugins\generic\inlineHtmlGalley\InlineHtmlGalleyBlockPlugin;
 
 abstract class InlineHtmlGalleySidebarBlockPlugin extends InlineHtmlGalleyBlockPlugin {
 
-    abstract function blockName();
+	abstract function blockName();
```
Comment (ctgraham) [inline InlineHtmlGalleySidebarBlockPlugin.php:20]: Spaces in this file were changed to tabs, but PKP has standardized on spaces.

**Review by ctgraham — CHANGES_REQUESTED** [review]
For PHP files, PKP has standardized to spaces rather than tabs, so we should keep that convention when possible.

Note that the bootstrap3 templates use tabs rather than spaces, so using tabs in the templates here is acceptable.

**Pull request merged**